fluentbit/conf/conf.go

```diff
@@ -668,6 +690,39 @@ func (s Syslog) renderConfig() (string, error) {
 	return renderedSyslogConfig.String(), nil
 }
 
+// A WindowsEventlog represents the configuration data for fluentbit's winlog input plugin
+type WindowsEventlog struct {
+        Tag string
+	Channels string
```

Comment:
Missing 2 spaces it seems. May want to set up `Run goimports` as instructed in the developer guide to use the tool to automatically adjust formatting.